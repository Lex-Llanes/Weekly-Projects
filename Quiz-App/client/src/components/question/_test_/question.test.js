import React, { createElement } from "react"
import ReactDOM from "react-dom";
import Question from "./../question"
import { render } from "@testing-library/react"
import { create } from "react-test-renderer"
import { isTSAnyKeyword } from "@babel/types"



describe("Question renders properly as specified", () => {
    it("Matches the snapshot", () => {
        //This will simulate calling the Question cxomponent (mounting it to the parent)
        const question = create(<Question />);
        //This is expectin git to match the component
        expect(question.toJSON()).toMatchSnapshot();
    })
})