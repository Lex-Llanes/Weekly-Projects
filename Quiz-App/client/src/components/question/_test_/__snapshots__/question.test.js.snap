// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question renders properly as specified Matches the snapshot 1`] = `
<div>
  <br />
  <h1>
    Quiz question is: 
  </h1>
  <h5
    dangerouslySetInnerHTML={
      Object {
        "__html": undefined,
      }
    }
  />
</div>
`;
